Only apply the constraint to the parent with the new syntax (#2117)
slint-ui/slint: docs/layouting.md

A value of `0` means that the element will not be stretched at all; unless all siblings also have a stretch
factor of `0`. Then all the elements will be equally stretched.

> The default value of these constraint property may depends on the content of the element.

Reviewer: ```suggestion
The default value of these constraint properties may depends on the content of the element.
```

If the element does not set a `x` or a `y` property, these constraints are also automatically applied to the parent element.

> (Unless using the legacy syntax, in which case it is only the layout element that apply their constraints to the parent)

Reviewer: ```suggestion
When using the legacy syntax, only the layout elements apply their constraints to the parent.
```
